packages/base/card-api.gts

```diff
@@ -223,7 +230,8 @@ export interface Field<
   component(
     model: Box<CardBase>,
     format: Format,
-    actions?: Actions
+    actions?: Actions,
+    context?: CardRenderingContext
```

Comment:
From our discussion, I was imagining `actions` as being a property on the CardRenderingContext.

Considering we are almost always passing actions AND context in this PR, I think this change would be a nice simplification.

Reply:
That makes sense. I'm working on consolidating both into the context argument.